integration-examples/splunk-otel-dotnet-docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
```

Comment:
Consider to upgrade to 8.0 (together with csproj and other places).

It will be longer valid example.

Reply:
https://github.com/splunk/observability-content-contrib/pull/52/commits/7d728a342c7a68d5d0c343369d6dffe1b1e2f0c5